### ISSUE TITLE
MOUSE_MODE_CAPTURED is not locking cursor to center of the screen

### ISSUE THREAD

**Issue by RadicalMGuy** [godotengine/godot#45498]
**Godot version:** 3.2.3.stable.mono.official

**OS/device including version:** MacOS 11.1

**Issue description:** 

When I set the mouse mode to captured, the documentation states that the cursor should be locked to the center of the screen. Instead, the cursor is locked to wherever its position was when the window is focused. For example, when I hit the play button to run the project the cursor is hidden, then left clicking removes focus from the game window and shows that the cursor was still locked where the play button is. If I restore focus to the game window, the cursor locks to wherever it was when the focus is restored. This means that if I focus the game window and the cursor was outside of the window, left-clicking will remove focus from the game window. This is causing me difficulty with creating 3D projects that have mouse-controlled camera movement.

**Steps to reproduce:**

Create an empty project, add a new Node as the scene root. Attach a script with the following code:

```
extends Node

func _ready():
	Input.set_mouse_mode(Input.MOUSE_MODE_CAPTURED)

func _input(event):
	if Input.is_action_just_pressed("ui_cancel"):
		if Input.get_mouse_mode() == Input.MOUSE_MODE_CAPTURED:
			Input.set_mouse_mode(Input.MOUSE_MODE_VISIBLE)
		else:
			Input.set_mouse_mode(Input.MOUSE_MODE_CAPTURED)
```

For me, running this project and then left-clicking will cause the game window to lose focus. Using the ESC key, I can see that toggling the mouse mode between visible and captured always results in the cursor being locked and hidden at whatever its current position is.

**Comment by bruvzg** [comment]
#42321 duplicate, should be fixed in `master` and `3.2.4 beta`.

**Comment by akien-mga** [comment]
You can test with https://downloads.tuxfamily.org/godotengine/3.2.4/beta6 (or rc1 published later today) to confirm the fix.

Closing as duplicate of #42321.
